atom/browser/native_window_mac.h

```diff
@@ -162,6 +162,8 @@ class NativeWindowMac : public NativeWindow,
 
   bool is_kiosk_;
 
+  bool was_fullscreen_;
```

Comment:
This should be initialized to `false` in https://github.com/watilde/electron/blob/9bdaf30d22d93f4c9c294ce114f0103b4073b47b/atom/browser/native_window_mac.mm#L621-L622